### ISSUE TITLE
Error with hset command

### ISSUE THREAD

**Issue by tannerjt** [koopjs/koop-cache-redis#11]
From what I can tell, the hset is correctly providing a string via `JSON.stringify(value)` for the `set` method, but I'm getting this error:

```bash
node_redis: Deprecated: The HSET command contains a argument of type Object.
This is converted to "[object Object]" by using .toString() now and will return an error from v.3.0 on.
Please handle this in your code to make sure everything works as you intended it to.
error:  ReplyError: ERR wrong number of arguments for HMSET
```

I'm running redis v5.3.0

**Comment by rgwozdz** [comment]
Attempting a fix with https://github.com/koopjs/koop-cache-redis/pull/12

**Comment by rgwozdz** [comment]
Fixed in v1.0.2
